console/client/src/utils/search-params.utils.ts

```diff
@@ -0,0 +1,6 @@
+export const urlSearchParamsToObject = (params: URLSearchParams) =>  {
+  return [ ...params.entries() ].reduce((acc, [ key, value ]) => {
+    acc[key] = value
+    return acc
+  }, {})
+}
```

Comment:
You might be able to use here:

```ts
export const urlSearchParamsToObject = (params: URLSearchParams) => Object.fromEntries(params)
```

If this is similar we can maybe remove this helper/utils.

Reply:
Yep that works